src/lib/indexer/eventProcessor.ts

```diff
@@ -238,6 +250,71 @@ const DefaultEventProcessors: EventProcessors = {
       ]),
     }
   },
+
+  'match2-cancel': (version, _transaction, _sender, inputs, _outputs) => {
+    if (version !== 1) {
+      throw new Error(`Incompatible version ${version} for match2-cancel process`)
+    }
+
+    const demandALocalId = inputs[0].localId
+    const demandAId = _outputs[0].id
+    const demandBLocalId = inputs[1].localId
+    const demandBId = _outputs[1].id
+    const matchLocalId = inputs[2].localId
+    const matchId = _outputs[2].id
+    const match2Cancel = _outputs[2]
+
+    if (_transaction) {
```

Comment:
This still isn't right. We'll need to update the comment as part of this. Have a look at `demand-comment` for the logic. Also can we please rename the `_` variables when we're using them. That's just to document that they are unused intentionally bu tnow they are used

Reply:
thanks!